test/parallel/test-http-outgoing-implicit-header.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+require('../common');
+const http = require('http');
+const { strictEqual } = require('assert');
+
+const server = http.createServer((req, res) => {
+  if (req.method === 'HEAD') {
+    res.writeHead(200, { 'Content-Type': 'text/plain' });
+    res.end();
+  } else {
+    res.writeHead(200, { 'Content-Type': 'text/plain' });
+    res.end('Hello World\n');
+  }
+});
+
+server.listen(0, () => {
+  const port = server.address().port;
+  const options = {
+    method: 'GET',
```

Comment:
I think we want to verify that `HEAD` can't write. So I am not sure why we are only testing `GET` here.

Reply:
I did it to see if there is a problem with the general api 🤔